packages/react-components/react-aria/library/src/useTypingAnnounce/useTypingAnnounce.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { useTimeout } from '@fluentui/react-utilities';
```

Comment:
I'm not especially concerned about this here but I've been working with timers where this does matter so I figure I'll mention it and you can decide if it matters or not :)

When the window/tab is backgrounded timers are throttled. The exact throttle amount is browser-specific but it seems to be that they are called at most once per second. Since users are interacting directly with the page when these timers are in play, throttled timers seem like an edge case for this feature but perhaps a quick test is called for just to make sure nothing goes super sideways?

Reply:
live regions also should not do anything when a window/tab is in the background, so it should be fine, I believe